bazel/go_proto_library.bzl.diff

```diff
@@ -14,16 +14,16 @@ index 534064e..78aad52 100644
 +# 3. Support use_default_shell_env when working with non-default toolchain locations.
 +
  """A basic go_proto_library.
- 
+
```

Comment:
Is there some weird whitespace thing going on in this diff of the diff?

Reply:
looks like Intellij might have scrubbed the extra whitespace. I can add it back in if you think it's worth it.